### PR TITLE
generator extract as plugin

### PR DESCRIPTION
**Pull request by khus29**
**Updates Done**
1. Package json updated - created bin (component-generator)
2. Added file index js to trigger plop command
3. Updated relative path of atoms, molecules, organisms...

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+const args = process.argv.slice(2);
+const {Plop, run} = require('plop');
+const argv = require('minimist')(args);
```
Comment (vinodloha) [inline index.js:5]: minimist package is not available in dependencies.

**Comment by vinodloha** [comment]
@khus29 Are you going to add these features gradually? 

1. Ability to create components in mono-repo? 
2. TS or JS support.

File: index.js (hunk 1 above)
Comment (khus29) [reply index.js:5]: It is covered in this PR - https://github.com/pagesource/component-generator/pull/3/files

**Comment by khus29** [comment]
Updates i this PR included in the latest PR

**Pull request closed without merging**